tools/disk_speed_test.py

```
@@ -24,10 +24,14 @@ def disk_speed_test():
         print(f"Error: {e}")
 
+def check_platform_compatibility():
+    supported = True
+    warnings = []
 
-def main():
-    disk_speed_test()
-
+    try:
+        import psutil
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)